src/cmds/qsub.c

```diff
@@ -472,15 +472,29 @@ expand_varlist(char *varlist)
 	char *p1, *p2, *p;
 	char *ev;
 	int  v_value1_sz=0;
+	char *pc;
+	int special_char_cnt = 0;
+	int len = 0;
```

Comment:
let us align all the variable names to start in the same column.

Reply:
Implemented. But going forward we should have consistency by following either ONLY space or TAB for alignment.  But I do see we are using spaces sometimes and also tabs at some other places.